blockly-ws-search/example/web-script/package.json

```diff
@@ -2,8 +2,18 @@
   "name": "blockly-ws-search-web-script",
   "version": "1.0.0",
   "description": "Example of workspace search with script tag.",
+  "private": true,
+  "scripts": {
+    "preinstall": "npm install ../../../blockly-ws-search",
```

Comment:
If this is already in the devDependencies why do we need it here?

Reply:
`blockly-ws-search` is a dependency, but without this `preinstall`, `blockly-ws-search` isn't installed **before** this module causing an error "no such file or directory" (but the npm install works the second time you call it).
Adding this line ensures that npm install works the first time. I think this is only a problem only because `blockly-ws-search` is a file path.

Reply:
I don't understand this.  Can you explain further or link to a reference you looked at?